src/main/kotlin/no/nav/yrkesskade/saksbehandling/graphql/client/pdl/PdlException.kt

```diff
@@ -0,0 +1,5 @@
+package no.nav.yrkesskade.saksbehandling.graphql.client.pdl
+
+class PdlException : Exception {
+    constructor(message: String?) : super(message)
```

Comment:
Dersom du bare har en constructor, ville du lagt den som f.eks PdlException(message: String?) : Exception(message)? Spør for en venn.

Reply:
Jeg spurte min venn IntelliJ, som svarte ja 😄 jeg endrer både denne og SafException